src/main/resources/db/migration/V1_2__visit_id.sql

```diff
@@ -0,0 +1,9 @@
+create table visit_id
+(
+    nomis_id bigint       not null  constraint visit_id_nomis_id_pkey   primary key,
+    vsip_id  varchar(80)  not null  constraint visit_id_vsip_id_unique  unique,
+    label    varchar(20),
+    source   varchar(20)  not null
+);
```

Comment:
didn't @andymarke suggest adding a when created here too?

Reply:
just realised that `label` might be the created date

Reply:
thats right, im keeping it generic for now